README.md

```diff
@@ -11,7 +11,11 @@ A running instance of Postgres DB. The service connects to `postgres://sdx@sdxlo
 
 ## Installation
 
-Using virtualenv and pip, create a new environment and install within using:
+Using virtualenv and pip, create a new environment and install the sdx-common library from a local directory:
```

Comment:
Can we change this so that it shows the use of the makefile? Look at sdx-receipt-ctp for an example.